dcos_test_utils/helpers.py

```diff
@@ -205,7 +205,10 @@ def session_tempfile(data):
     the temp file will be removed when the interpreter exits
     """
     with tempfile.NamedTemporaryFile(delete=False) as f:
-        f.write(data)
+        if isinstance(data, str):
+            f.write(data.encode())
```

Comment:
With encode, it's always better to provide an encoding ( pref:`utf-8`) - otherwise, it picks up the platform default encoding which puts us to the mercy of package/OS manager setting proper environment variables.

Reply:
So, data.encode should be `data.encode('utf-8')`